docker/Dockerfile

```diff
@@ -0,0 +1,89 @@
+FROM debian:stretch as conpot-builder
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# Add non-free packages, needed for snmp-mibs-downloader
+RUN sed -i -e 's/main/main non-free contrib/g' /etc/apt/sources.list
+
+# Install dependencies
+RUN apt-get update -y -qq && apt-get install --no-install-recommends -y -qq \
+        ipmitool \
+        supervisor \
+        tcpdump \
+        git \
+        python3-pip \
+        python3 \
+        python3-dev \
+        python3-setuptools \
+        build-essential \
+        python3-scapy \
+        python3-cffi \
+        wget &&\
+    apt-get clean && \
+    rm -rf /var/lib/apt/lists/* /tmp/* /var/tmp/*
+
+# Copy the app from the host folder (probably a cloned repo) to the container
+RUN useradd -m conpot
+WORKDIR /opt/
+RUN git clone https://github.com/mushorg/conpot.git
+RUN chown conpot:conpot -R /opt/conpot
+
+# Install Python requirements
+USER conpot
+ENV PATH=$PATH:/home/conpot/.local/bin
+WORKDIR /opt/conpot
+RUN pip3 install --user --no-cache-dir coverage
+RUN pip3 install --user --no-cache-dir -r requirements.txt
+
+# Run test cases
+RUN py.test -v
+
+# Install the Conpot application
+RUN python3.5 setup.py install --user --prefix=
+
+WORKDIR /home/conpot
+RUN tar -zcf conpot.tgz .local
+
+FROM debian:stretch
+
+ENV DEBIAN_FRONTEND noninteractive
+USER root
+RUN sed -i -e 's/main/main non-free contrib/g' /etc/apt/sources.list
+
+WORKDIR /home/conpot
+
+RUN apt-get update -y -qq && apt-get install --no-install-recommends -y -qq \
+        ipmitool \
+        ca-certificates \
+        tcpdump \
+        python3 \
+        python3-scapy \
```

Comment:
`scapy` and `cffi` are installed as part of requirements. 

https://github.com/mushorg/conpot/blob/ce943bb852abfa931acd3112033e5c78092be1fb/requirements.txt#L16

and 

https://github.com/mushorg/conpot/blob/ce943bb852abfa931acd3112033e5c78092be1fb/requirements.txt#L2